### ISSUE TITLE
Wonderful job, would like to ask a question

### ISSUE THREAD

**Issue by lxr2** [can-can-ya/QPMIL-VL#1]
After reading your paper, I found the content very insightful. However, I have a question I hope you can clarify.  

In Figure 4(a) of your paper, you mentioned that "the 5 keys predominantly matched by different datasets are often non-overlapping due to the matching penalty." How is this achieved? Specifically, in Task 1, shouldn't the model uniformly sample all prototypes instead of the only 5 keys? Thank you!

**Comment by can-can-ya** [comment]
Hi, @lxr2, thanks for your attention.

When the model is trained on the first task, although no matching penalty is applied, experimental analysis shows that all samples in the first task tend to be matched to the same group of keys during training. When the model is incrementally trained on subsequent tasks, if no matching penalty is added, the samples from these tasks also tend to be matched to the same group of keys, leading to severe catastrophic forgetting. Therefore, we introduce a matching penalty during the training of subsequent tasks to alleviate this problem. A detailed explanation of this process can be found in the Prototype Pool section of the method description in the paper.

**Comment by lxr2** [comment]
Many thx, do you have any schedule to release the code?

**Comment by can-can-ya** [comment]
> Many thx, do you have any schedule to release the code?

Our source code will be publicly available after the paper is accepted.

**Comment by lxr2** [comment]
Great, another question is in your paper, it mentions "We employ the classic ABMIL (Ilse, Tomczak, and Welling 2018) as the aggregator for the vision branch,serving as a **Baseline**." in the experiments section. I don't understand it since QPMIL-VL directly uses Eq. (3) **prototype-guided aggregation** to get the bag-level feature, right? So where do you use ABMIL?

**Comment by can-can-ya** [comment]
Yes, **for QPMIL-VL**, we utilize the **Prototype Pool** and **Prototype-guided Aggregation** modules to derive WSI bag-level feature.

However, **for other comparative incremental methods**, *e.g.*, Baseline, Regularization-based, and Rehearsal-based methods, instance aggregation is performed via **ABMIL**, as the Prototype Pool and Prototype-guided Aggregation are integral parts of our designed incremental learning framework.

Specifically, **for Rehearsal-based methods**, in the vision branch, we perform instance aggregation through **ABMIL**, while in the language branch, we directly use the **Tumor Type** text descriptions from Tab. 5 and obtain the final class feature via the CONCH text encoder. Additionally, **specific rehearsal strategies** are applied to mitigate catastrophic forgetting.

**Comment by lxr2** [comment]
Cool! Thank you for the detailed explanation!

**Comment by can-can-ya** [comment]
> Many thx, do you have any schedule to release the code?

Hi, code is live now~
